testing/tests/named_end.rs

```diff
@@ -0,0 +1,13 @@
+use askama::Template;
```

Comment:
Let's just put this in `inheritance.rs` rather than adding yet another file?

Reply:
Don't you prefer to have a specific test for it? I'm not against it, just want to be sure before making the change.

Reply:
Ok!